pkg/printer/print.go

```diff
@@ -527,12 +527,23 @@ func ConfigServiceTokens(tokens []models.ConfigServiceToken, number int, jsonFla
 	Table([]string{"name", "slug", "created at"}, rows, TableOptions())
 }
 
-// ConfigServiceToken print config service token
+// ConfigServiceTokenInfo print config service token info
+func ConfigServiceTokenInfo(token models.ConfigServiceToken, jsonFlag bool, diff bool) {
+	if jsonFlag {
```

Comment:
Is there a way we can add a --plain option or some flag just to return the token?

Reply:
I like it!